src/http-common.ts

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+
+export default axios.create({
+    // TODO: change baseURL
+    baseURL: "http://localhost:8080/api",
```

Comment:
You should use the `proxy` config in `package.json` instead!

Reply:
will do 👍